src/mixins/createStatefulChildrenMixin.ts

```diff
@@ -170,7 +170,7 @@ function manageChildren(evt: StateChangeEvent<StatefulChildrenState>): void {
 			}
 			else {
 				/* Tuple of Promise, child ID, position in child list */
-				resolvingWidgets.push([ internalState.registry.get(id), id, key ]);
+				resolvingWidgets.push([ internalState.registry.has(id).then((exists) => { if (exists) { return internalState.registry.get(id); } }), id, key ]);
```

Comment:
`get` rejects the promise and therefore we get into the error handling on the `Promise.all` of the `resolvingWidgets` - we do not want that to happen. So the has check determines if we should try and get the widget from the registry.